source/basic.tex

```diff
@@ -2680,6 +2687,14 @@
 they both declare names with external linkage.
 \end{itemize}
 \begin{note}
+An \grammarterm{alias-declaration}\iref{dcl.typedef},
```

Comment:
Why not add this note instead to [basic.pre]/5 (changed above), perhaps phrased as

> A *typedef-name* or *namespace-alias* is not itself an entity.  A *using-declarator* does not introduce an entity.

?  I think *namespace-alias-definition* is then covered by implication.  (The non-parallelism in that wording results from the lack of a pseudo-grammar term like *using-name* to distinguish identifiers with such a declaration, which I regard as no great omission.)

If we prefer to leave it here, I think it goes better after the "other circumstances" note (which is more directly related to the paragraph).